Makefile

```diff
@@ -389,7 +394,10 @@ endif
 	# Copy in system image build script
 	$(INSTALL_M) $(JULIAHOME)/contrib/build_sysimg.jl $(DESTDIR)$(datarootdir)/julia/
 	# Copy in all .jl sources as well
-	cp -R -L $(build_datarootdir)/julia $(DESTDIR)$(datarootdir)/
+	mkdir -p $(DESTDIR)$(datarootdir)/julia/{base,test}
+	cp -R -L $(JULIAHOME)/base/* $(DESTDIR)$(datarootdir)/julia/base
+	cp -R -L $(JULIAHOME)/test/* $(DESTDIR)$(datarootdir)/julia/test
```

Comment:
Why are you explicitly copying `base` and `test` here?  The `cp -R -L $(build_datarootdir)/julia/* $(DESTDIR)$(datarootdir)/julia` line should do this automatically.

Reply:
For out of tree builds, `$(build_datarootdir)/julia` contains two symlinks: base and test. The problem is that the `base` symlink points at the `base` directory in the build tree e.g. `build/usr/share/julia/base -> build/base`.  The effect is that `cp -R -L` only copies the six or so build-time generated files in `build/base` and misses all of the files in `$JULIAHOME/base`.  This chunk is intended to fix that.